src/openms/include/OpenMS/ANALYSIS/MRM/ReactionMonitoringTransition.h

```diff
@@ -78,13 +79,17 @@ namespace OpenMS
 
     const String & getNativeID() const;
 
-    void setPeptideRef(const String & peptide_ref);
+    /// Sets the transition reference (unified method)
+    void setTransRef(const String & transition_ref, OpenSwath::TransType type);
```

Comment:
Maybe rename to setTransGroupRef.

Transitions refer to the individual MS2 traces which might be confusing since here it is referring to a group of transitions associated with a compound/peptide/nucleotide